.buildkite/scripts/steps/console_definitions_sync.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+GENERATED_DEFINITIONS="src/plugins/console/server/lib/spec_definitions"
```

Comment:
I executed the pipeline manually with script changes that havent been commited yet, so these will be gone after that is merged